.github/ISSUE_TEMPLATE/add-orbit-chain-request.yml

```diff
@@ -105,11 +106,12 @@ body:
       placeholder: ex. ETH
     validations:
       required: false
-  - type: input
+  - type: textarea
     id: native-token-logo
     attributes:
       label: Native token logo
-      description: Link to the native token logo file.
+      description: Please drag and drop your native token logo image here. Supported formats: PNG, SVG, JPG, JPEG, WEBP. Images larger than 100KB will be resized.
```

Comment:
I'm not sure if we need it, but I can imagine a scenario where teams would upload ETH logo here if they use it as the native currency 😆 So maybe we should mention not to do that for ETH?